### ISSUE TITLE
"ignoring prop" warnings?

### ISSUE THREAD

**Issue by mikecann** [inlet/react-pixi#235]
# Description

I am getting a bunch of these warning `ignoring prop: mask, from null to null for AnimatedSprite`

![chrome_2020-06-27_16-29-46](https://user-images.githubusercontent.com/215033/85918269-c56a4200-b893-11ea-94f7-aea684a2c442.png)

Im not sure why or what to do about them.

### Steps to reproduce

The code looks like this:

```typescript
import * as React from "react";
import { Point2D } from "@markd-co/battletabs-shared";
import { Container, AnimatedSprite } from "@inlet/react-pixi";
import * as PIXI from "pixi.js";
import { ShipSegmentMask } from "./ShipSegmentMask";
import { AnimatedSpriteWithPixelPerfectCollisions } from "./AnimatedSpriteWithPixelPerfectCollisions";
import { withPixelPerfectCollisions } from "./HOCs/withPixelPerfectCollisions";

interface Props extends React.ComponentProps<typeof AnimatedSprite> {
  pivot: Point2D;
}

const Sprite = withPixelPerfectCollisions(AnimatedSprite);

export const ShipSegment: React.FC<Props> = React.memo(({ pivot, ...rest }) => {
  const [mask, setMask] = React.useState<PIXI.Graphics | null>(null);

  return (
    <>
      <ShipSegmentMask ref={setMask} />
      <Sprite
        hitmapResourceId={"vikingShipsHitmap"}
        mask={mask}
        pivot={pivot}
        {...rest}
      />
    </>
  );
});
```

### Additional info

- `@inlet/react-pixi` version: 5.0.0
- `React` version: 16.13.0
- `ReactDOM` version: 16.13.0
- `PIXI` version: 5.3.0

**Comment by inlet** [comment]
Hi @mikecann 

Good point, a `null` value is a possible value for prop `mask`. Right now we prompt these warnings to tell the user a possible invalid value is set. At runtime we don’t know the valid types for a prop. 

I can create a PR which bypass the mask null value.

**Comment by mikecann** [comment]
Cool, ill take a look at your PR once you have done it because I get a bunch of these warnings for other valid values. Ill submit PR's for these values when I come across them.

**Comment by inlet** [comment]
I've pushed a patch. It's now fixed in `v5.0.1`
